integration/src/integration/java/pl/allegro/tech/hermes/integration/management/SubscriptionManagementTest.java

```diff
@@ -127,6 +130,43 @@ public void shouldUpdateSubscriptionEndpoint() throws Throwable {
         remoteService.waitUntilReceived();
     }
 
+    @Test
+    public void shouldUpdateSubscriptionPolicy() {
```

Comment:
I had problem with updating topic with `sendingDelay` so I created test for it. I turned out that I forgot to change hashCode and equals. I decided that I could just leave this test, it might be useful in future.